### PR TITLE
test(app): add tests for CheckboxField

### PR DESCRIPTION
**Pull request by koji**
# Overview
This will close RAUT-211

# Changelog
- modify CheckboxField (destruct props, remove unexpected null)
- add test cases

# Review requests
- [ ] test covers basic cases
- [ ] If anyone has an idea to test OUTER_STYLE (currently comment out)

# Risk assessment
low

### CODE REVIEW

File: app/src/atoms/CheckboxField/__tests__/CheckboxField.test.tsx [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+    // getByLabelText('ot-checkbox')
+
+    // label OUTER_STYLE
+    // expect(checkBoxLabel).toHaveStyle('@apply --font-form-default')
+    // expect(checkBoxLabel).toHaveStyle('font-size: 0.75rem')
+    // expect(checkBoxLabel).toHaveStyle('font-weight: 400')
+    // expect(checkBoxLabel).toHaveStyle(`color: ${COLORS.darkBlackEnabled}`)
+    // expect(checkBoxLabel).toHaveStyle('display: flex')
+    // expect(checkBoxLabel).toHaveStyle(`align-items: ${ALIGN_CENTER}`)
+    expect(checkBoxLabel).toHaveStyle('line-height: 1')
```
Comment (koji) [inline app/src/atoms/CheckboxField/__tests__/CheckboxField.test.tsx]: Devtools show `OUTER_STYLE` but this test code doesn't work as expected

File: app/src/atoms/CheckboxField/index.tsx [hunk 2]
```diff
@@ -104,32 +112,47 @@ const LABEL_TEXT_STYLE = css`
...
+    tabIndex = 0,
+    isIndeterminate,
+  } = props
+  const indeterminate = isIndeterminate ?? false ? 'true' : undefined
 
   return (
-    <label css={OUTER_STYLE}>
-      {props.isIndeterminate ? (
-        <Icon name="minus-box" width="100%" css={INNER_STYLE_VALUE} />
+    <label css={OUTER_STYLE} data-testid="CheckboxField_label">
```
Comment (koji) [inline app/src/atoms/CheckboxField/index.tsx]: test code doesn't get this `OUTER_STYLE` even though I added `data-testid`

**Comment by koji** [comment]
@Opentrons/app-and-uis test

**Review by brenthagen — APPROVED** [review]
✔️

**Pull request merged**